intermediate_html_css/intermediate_html_concepts/introduction.md

```diff
@@ -1,20 +1,20 @@
 ### Introduction
 
-This is the first of the two part course where we dig deeper into HTML and CSS.
+This is the first of the two-part course where we dig deeper into HTML and CSS.
 
-Our foundations lessons by design only scratched the surface of what's possible. We wanted to give you all of the bare necessities for getting started so you could move on and be somewhat productive as quickly as possible, but now it's time to slow down and dig in!
+Our foundation lessons by design only scratched the surface of what's possible. We wanted to give you all of the bare necessities for getting started so you could move on and be somewhat productive as quickly as possible, but now it's time to slow down and dig in!
```

Comment:
This is referring to our Foundations course, so perhaps rephrasing the whole sentence would be better. 
```
The lessons in our Foundations course only scratched the surface of what's possible...
```